Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -22,6 +22,12 @@
   "loc.input.help.Package": "Folder or file path to the App Service package. Variables ( [Build](https://www.visualstudio.com/docs/build/define/variables) | [Release](https://www.visualstudio.com/docs/release/author-release-definition/understanding-tasks#predefvariables)), wild cards are supported. <br/> For example, $(System.DefaultWorkingDirectory)/\\*\\*/\\*.zip.",
   "loc.input.label.WebAppUri": "App Service URL",
   "loc.input.help.WebAppUri": "Specify a name for the output variable that is generated for the URL of the Azure App Service. The variable can be consumed in subsequent tasks.",
+  "loc.input.label.AddWebConfig": "Add web.config",
+  "loc.input.help.AddWebConfig": "Select this option to generate web.config file for your node app",
+  "loc.input.label.AppType": "App type",
```

Comment:
I am not so sure we should use short for Application in the label. it should say "Application type"